cmd/cluster-node-tuning-operator/main.go

```diff
@@ -134,6 +136,10 @@ func operatorRun() {
 		klog.Exit(err)
 	}
 
+	if err := setCgroupsV1(restConfig); err != nil {
```

Comment:
This will put all clusters into V1 mode, irrespective of a performance profile presence... The operator is always present.

Reply:
So this must be at the reconcile level